src/components/UI/DropdownMenu.js

```diff
@@ -61,12 +61,12 @@ const MenuWrapper = styled.div`
 `;
 
 function DropdownMenu(props) {
-  const [isOpen, setIsOpen] = useState(true);
+  const [isOpen, setIsOpen] = useState(false);
   let timeOutId = null;
 
   const onBlurHandler = () => {
     timeOutId = setTimeout(() => {
-      setIsOpen(true);
+      setIsOpen(false);
     });
```

Comment:
What should this timeout do? It has no timeout period

Reply:
IIRC this is to put this call at the end of the queue, this way we can first check if any child has focus before closing the menu. This is an old JS hack.

Reply:
If you don't specify time, it is the same as to say "on the next tick", check it out: https://reactjs.org/docs/accessibility.html

Reply:
I know what setTimeout(..., 0) does, but I didn't know that `0` was the default value of `setTimeout` 👍